manifests/params.pp

```diff
@@ -26,6 +26,10 @@
   $puppet_ca_proxy               = ''
 
   $foreman_proxy_port            = '9090'
+
+  $foreman_proxy_http            = true
+  $foreman_proxy_http_port       = '8000'
```

Comment:
Yes - it's the foreman default.  I imagine it could be changed to be `$foreman_proxy::params::http_port`.

```
[root@mahcapsule ~]# setenforce 1
[root@mahcapsule ~]# service foreman-proxy restart
Stopping foreman-proxy:                                    [  OK  ]
Starting foreman-proxy:                                    [  OK  ]
[root@mahcapsule ~]# telnet localhost 8000
Trying ::1...
telnet: connect to address ::1: Connection refused
Trying 127.0.0.1...
Connected to localhost.
Escape character is '^]'.
```